.clang-format

```diff
@@ -21,6 +21,57 @@ BreakConstructorInitializers: BeforeComma
 BreakInheritanceList: BeforeComma
 ColumnLimit: '100'
 FixNamespaceComments: 'true'
+IncludeBlocks: Regroup
+IncludeCategories:
+
+  # Desired final ordering:
```

Comment:
@elrond79 You copy-pasted from wrong clang-format file, i.e. this doesn't match what we have below in coding guidelines document.